asciidoc-confluence-publisher-cli/src/main/java/org/sahli/asciidoc/confluence/publisher/cli/AsciidocConfluencePublisherCommandLineClient.java

```diff
@@ -51,6 +52,9 @@ public static void main(String[] args) throws Exception {
         String spaceKey = mandatoryArgument("spaceKey", args);
         String ancestorId = mandatoryArgument("ancestorId", args);
 
+        String publishStrategyParam = optionalArgument("strategy", args).orElse(null);
```

Comment:
You could already pass `APPEND_TO_ANCESTOR.name()` as the default value when retrieving the publish strategy and then avoid the subsequent `null`-check on the next line.